packages/react-charting/package.json

```diff
@@ -45,7 +45,7 @@
     "@types/d3-sankey": "^0.11.0",
     "@types/d3-scale": "^4.0.0",
     "@types/d3-selection": "1.4.1",
-    "@types/d3-shape": "2.1.0",
+    "@types/d3-shape": "3.1.1",
```

Comment:
Why are @types/d3-shape and d3-shape of different version

Reply:
@types/d3-hape highest available version is 3.1.1. And whatever change we are looking for d3-shape is available in 3.2.0. So i have just upgraded the types to highest available version.